xml/System.Numerics/Complex.xml

```diff
@@ -1290,9 +1355,9 @@
         <Parameter Name="value" Type="System.Numerics.Complex" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">A complex number.</param>
+        <summary>Determines whether the specified complex number is finite.</summary>
+        <returns><see langword="true"> if both, the real and imaginary parts are finite (zero, subnormal or normal); otherwise, <see langword="false">.</returns>
```

Comment:
Build failure:
```suggestion
        <returns><see langword="true" /> if both, the real and imaginary parts are finite (zero, subnormal or normal); otherwise, <see langword="false" />.</returns>
```

Reply:
I just pushed that change, thanks for catch it.